TAF/testCaseModules/keywords/performance-metrics-collection/RetrieveResourceUsage.py

```diff
@@ -173,13 +174,23 @@ def get_service_mem_aggregation_value(service, resource_list):
 
 def calculate_memory_usage(d):
     memory_usage = 0
+
     try:
         if "cache" in d["memory_stats"]["stats"]:
-            memory_usage = d["memory_stats"]["usage"] - d["memory_stats"]["stats"]["cache"]
+            try:
+                memory_usage = d["memory_stats"]["usage"] - d["memory_stats"]["stats"]["cache"]
+            except:
+                logger.error("Fail to calculate memory usage by usage - cache")
```

Comment:
may we remove the try catch and use `logger.info("Calculating memory usage by usage - cache")`?

Reply:
Update to use logger.info, and it will always print the log.